multinet/uploaders/csv.py

```diff
@@ -57,16 +57,46 @@ def flask_response(self) -> FlaskTuple:
         return ("Could not read CSV data", "415 Unsupported Media Type")
 
 
-def validate_csv(
-    rows: Sequence[MutableMapping], key_field: str = "_key", overwrite: bool = False
-) -> None:
-    """Perform any necessary CSV validation, and return appropriate errors."""
+def is_edge_table(rows: Sequence[MutableMapping]) -> bool:
+    """Determine if this table should be treated as an edge table."""
+    fieldnames = rows[0].keys()
+    return "_from" in fieldnames and "_to" in fieldnames
+
+
+def is_node_table(
+    rows: Sequence[MutableMapping], key_field: str, overwrite: bool
```

Comment:
Why does this function take the `overwrite` parameter?

Reply:
Good catch, it shouldn't be there